homeassistant/components/fibaro/sensor.py

```diff
@@ -56,10 +63,20 @@ def setup_platform(
     """Set up the Fibaro controller devices."""
     if discovery_info is None:
         return
+    entities = []
```

Comment:
You need to add a type hint here to pass mypy.
```suggestion
    entities: list[SensorEntity] = []
```

Reply:
Done